src/test/java/io/personium/jersey/engine/test/ScriptTestForDynamicPath.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Personium
+ * Copyright 2014 - 2017 FUJITSU LIMITED
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.personium.jersey.engine.test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+import io.personium.client.DaoException;
+import io.personium.client.http.PersoniumRequestBuilder;
+import io.personium.client.http.PersoniumResponse;
+import io.personium.jersey.engine.test.categories.Integration;
+
+/**
+ * User script test.
+ */
+@RunWith(PersoniumEngineRunner.class)
+@Category({Integration.class })
+public class ScriptTestForDynamicPath extends ScriptTestBase {
+    /**
+     * constructor.
+     */
+    public ScriptTestForDynamicPath() {
+        super("io.personium.engine");
+    }
+
+    /**
+     * generate request url for executing via personium-core
+     * @param relativePath engine reletive path in service collection
+     */
+    protected String requestUrlForIntegrationTest(final String relativePath) {
+        return String.format("%s/%s/%s/%s/%s?cell=%s", baseUrl, cellName, boxName, serviceCollectionName, relativePath, cellName);
+    }
+
+    /**
+     * Testing whether dynamic path is available
+     */
+    @Test
+    public final void dynamicPathAvailableTest() {
+        if (!isServiceTest) return;
+        putScript("cell.js", "test.js");
+        try {
+            configureService("{id}/test", "test.js");
+        } catch(DaoException e) {
+            fail(e.getMessage());
+        }
+        String url = requestUrlForIntegrationTest("hogehoge/test");
+        callServiceTest(url);
+        delScript("test.js");
```

Comment:
Please unify file deletion policy in all test method. For example, adding try-finally like `dynamicPathWithIllegalRoutePatternReturns503` method.